packages/ui/src/theme/tokens/components/card.ts

```diff
@@ -1,4 +1,43 @@
-export const card = {
+import {
+  BackgroundColorValue,
+  BorderColorValue,
+  BorderRadiusValue,
+  BorderStyleValue,
+  BorderWidthValue,
+  BoxShadowValue,
+  DesignToken,
+  SpaceValue,
+} from '../types/designToken';
+
+interface OutlinedTokens {
+  backgroundColor: DesignToken<BackgroundColorValue>;
+  borderRadius: DesignToken<BorderRadiusValue>;
+  borderWidth: DesignToken<BorderWidthValue>;
+  borderStyle: DesignToken<BorderStyleValue>;
+  borderColor: DesignToken<BorderColorValue>;
+  boxShadow: DesignToken<BoxShadowValue>;
+}
+interface ElevatedTokens {
+  backgroundColor: DesignToken<BackgroundColorValue>;
+  borderRadius: DesignToken<BorderRadiusValue>;
+  borderWidth: DesignToken<BorderWidthValue>;
+  borderStyle: DesignToken<BorderStyleValue>;
+  borderColor: DesignToken<BorderColorValue>;
+  boxShadow: DesignToken<BoxShadowValue>;
+}
+export interface CardTokens {
+  backgroundColor: DesignToken<BackgroundColorValue>;
+  borderRadius: DesignToken<BorderRadiusValue>;
+  borderWidth: DesignToken<BorderWidthValue>;
+  borderStyle: DesignToken<BorderStyleValue>;
+  borderColor: DesignToken<BorderColorValue>;
+  boxShadow: DesignToken<BoxShadowValue>;
+  padding: DesignToken<SpaceValue>;
+  outlined: OutlinedTokens;
+  elevated: ElevatedTokens;
+}
```

Comment:
This could probably be simplified I think. What these types should really say: the default and all variations (elevated and outlined) have the same tokens. Maybe something like:

```ts
interface CardVariationTokens {
  backgroundColor: DesignToken<BackgroundColorValue>;
  borderRadius: DesignToken<BorderRadiusValue>;
  borderWidth: DesignToken<BorderWidthValue>;
  borderStyle: DesignToken<BorderStyleValue>;
  borderColor: DesignToken<BorderColorValue>;
  boxShadow: DesignToken<BoxShadowValue>;
}

type CardTokens = CardVariationTokens & {
  outlined: CardVariationTokens;
  elevated: CardVariationTokens;
}
```

Reply:
Good call, updated.